config/connect-distributed.properties

```diff
@@ -75,6 +75,9 @@ offset.flush.interval.ms=10000
 #rest.advertised.host.name=
 #rest.advertised.port=
 
+# Controls presence of WADL information in a response to OPTIONS request
+#rest.wadl.enable=false
+
```

Comment:
Do we also want to include this in the `connect-standalone.properties`?

Reply:
Good catch @rhauch , thanks. I've included new config into the standalone config file.